UA Reference.sln

```diff
@@ -41,6 +41,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Opc.Ua.Bindings.Https", "St
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Opc.Ua.Security.Certificates", "Libraries\Opc.Ua.Security.Certificates\Opc.Ua.Security.Certificates.csproj", "{4B72937F-5A57-4CEA-B9FE-B4C45CF7B284}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Quickstarts.Servers", "Applications\Quickstarts.Servers\Quickstarts.Servers.csproj", "{83CC0D9C-40CD-487A-BF45-C4A8C1B8BF69}"
```

Comment:
Maybe move the Quickstarts.Servers project inside Applications folder?

![image](https://user-images.githubusercontent.com/13765819/124600697-aa032100-de6f-11eb-878e-674e0f581810.png)

Reply:
good point..

Reply:
followed starter kit with 'Stack' as well